sql/2020/14_PWA/top_manifest_display_values.sql

```diff
@@ -0,0 +1,38 @@
+#standardSQL
+# Top manifest display values - based on 2019/14_04c.sql
+CREATE TEMPORARY FUNCTION getDisplay(manifest STRING)
+RETURNS STRING LANGUAGE js AS '''
+try {
+  var $ = JSON.parse(manifest);
+  if (!('display' in $)) {
+    return '(not set)';
+  }
+  return $.display;
+} catch (e) {
+  return null;
+}
+''';
+
+SELECT
+  client,
+  getDisplay(body) AS display,
```

Comment:
If this is case insensitive
```suggestion
  LOWER(getDisplay(body)) AS display,
```

Reply:
Good spot as there were mixed cases in results. Fixed.